src/main/groovy/com/redpillanalytics/gradle/ConfluentPluginExtension.groovy

```diff
@@ -70,6 +70,16 @@ class ConfluentPluginExtension {
     */
    String pipelineEndpoint = 'http://localhost:8088'
 
+   /**
+    * Username for Basic Authentication with the RESTful endpoint. Default: ''.
+    */
+   String pipelineUsername = ''
```

Comment:
Does this need to be changed to `String pipelineUsername` like it was in the KsqlRest.groovy file?

Reply:
I'm @Sikta. I had to create a work account for some stuff we are working on.

Reply:
> Does this need to be changed to `String pipelineUsername` like it was in the KsqlRest.groovy file?

Good catch.

Reply:
New release will be published in the next 20 minutes or so.

Reply:
I can't thank you enough! This plugin saved us a ton of work figuring out how to deploy KSQL in our org.